lms/templates/api_admin/catalog-results.underscore

```diff
@@ -0,0 +1,6 @@
+<h2 class="api-subheading"><%- gettext("This catalog's courses:") %></h2>
+<ul class="preview-results-list">
```

Comment:
This is a good point. We could give back a paginated response, but then we'd need to do a decent amount of front-end work to create the correct JS views. We have the paginated collection and views, but they work with page/page size pagination instead of the limit/offset that the catalog API uses, and also the view might be difficult to incorporate into the page. Maybe we should offer a preview of the first N courses and then a link to the discovery service to show the rest? @clintonb might also want to weigh in.